examples/style_sheets/plot_bmh.py

```diff
@@ -1,21 +1,31 @@
 """
-This example demonstrates the "bmh" style, which is the design used in the
-Bayesian Methods for Hackers online book.
+=====================
+Bayesian Methods for Hackers style sheet
+=====================
+
+This example demonstrates the style used in the Bayesian Methods for Hackers
+[1]_ online book.
+
+.. [1] http://camdavidsonpilon.github.io/Probabilistic-Programming-and-Bayesian-Methods-for-Hackers/
```

Comment:
Question to a PEP8 expert  (@QuLogic ?) : are URLs longer than 80 characters OK with our pycodestyle setup?